js/confidenceSliderModel.js

```diff
@@ -1,222 +1,209 @@
-        return;
-      }
+    this.set({
+      _items: items,
+      _marginDir: (Adapt.config.get('_defaultDirection') === 'rtl' ? 'right' : 'left')
+    });
+  }
 
+  /* override */
+  restoreUserAnswers() {
+    if (!this.get('_isSubmitted')) {
       this.set({
-        _showNumber: linkedModel.get('_showNumber'),
-        _showScaleIndicator: linkedModel.get('_showScaleIndicator'),
-        _showScale: linkedModel.get('_showScale'),
-        labelStart: linkedModel.get('labelStart'),
-        labelEnd: linkedModel.get('labelEnd'),
-        _scaleStart: linkedModel.get('_scaleStart'),
-        _scaleEnd: linkedModel.get('_scaleEnd')
+        _selectedItem: {},
+        _userAnswer: undefined
       });
+      return;
+    }
 
-      this.set('_linkedModel', linkedModel);
-
-      if (this.get('_attempts') < 0) linkedModel.set('_attempts', 1);
-    },
-
-    _getGenericFeedback: function() {
-      return this.get('_feedback').generic;
-    },
-
-    _getComparisonFeedback: function() {
-      var lm = this.get('_linkedModel'),
-          confidence = this.get('_selectedItem').value,
-          linkedConfidence = lm.has('_userAnswer') ? lm.get('_userAnswer') : lm.get('_selectedItem').value,
-          feedbackString;
-      if (linkedConfidence < confidence) {
-        feedbackString = this.get('_feedback')._comparison.higher;
-      } else if (linkedConfidence > confidence) {
-        feedbackString = this.get('_feedback')._comparison.lower;
-      } else {
-        feedbackString = this.get('_feedback')._comparison.same;
-      }
-      return feedbackString;
-    },
-
-    _getThresholdFeedback: function() {
-      var feedbackList = this.get('_feedback')._threshold;
-
-      if (!feedbackList) return;
-
-      var confidenceValue = this.get('_selectedItem').value;
+    // this is only necessary to avoid an issue when using adapt-devtools
+    if (!this.has('_userAnswer')) this.set('_userAnswer', this.get('_items')[0].value);
+
+    super.restoreUserAnswers(this, arguments);
+  }
+
+  /* override */
+  canSubmit() {
+    return (!this.has('_linkedModel') || this.get('_linkedModel').get('_isSubmitted'));
+  }
+
+  /* override */
+  setupFeedback() {
+    this.set({
+      feedbackTitle: this.get('title'),
+      feedbackMessage: this.getFeedbackString()
+    });
+  }
+
+  /* override */
+  updateButtons() {
+    if (this.get('_attempts') > 0) {
+      super.updateButtons(this, arguments);
+      return;
+    }
+    this.set('_buttonState', this.get('_isEnabled') ? 'submit' : 'reset');
+  }
+
+  updateTracking() {
+    const shouldTrackResponses = (this.get('_shouldStoreResponses'));
+    const isSpoorEnabled = (Adapt.config.has('_spoor')?._isEnabled);
+    const shouldStoreResponses = (Adapt.config.get('_spoor')?._tracking?._shouldStoreResponses);
+    if (!shouldTrackResponses && !isSpoorEnabled && shouldStoreResponses) return;
```

Comment:
I think these should be ORs, not ANDs

Reply:
I'm not sure we need this any more.... Spoor stores this data itself... I'm not sure what this is for... Why do we have a separate `_shouldStoreResponses` here?

Reply:
```js
// if spoor is handling response tracking we don't need to do anything
```
Comment as a clue...

Reply:
@chris-steele Can you explain the use-case here? If spoor was disabled or wasn't storing responses, where was the 
confidenceSlider offlineStorage data going?

Reply:
I beleive the original code was to allow us to store confidence slider responses when spoor was enabled but storage of responses were disabled

Reply:
Is just requiring `_shouldStoreResponses` a suitable alternative?

Reply:
Agreed to remove spoor dependency from this plugin and add `_shouldStoreResponses` requirement to readme.